pems/README.md

```diff
@@ -0,0 +1,47 @@
+  - **occupancy1 (double):** The fraction of each sample period during which the detector in lane 1 output is “1”.
+  - **speed1 (double):**  The average speed of vehicles in lane 1 during the sample period.
+  - **flow2 (int):**  The number of vehicles observed in lane 2 for the sample period.
+  - **occupancy2 (double):** The fraction of each sample period during which the detector in lane 2 output is “1”.
+  - **speed2 (double):**  The average speed of vehicles in lane 2 during the sample period.
+  - **flow3 (int):**  The number of vehicles observed in lane 3 for the sample period.
+  - **occupancy3 (double):** The fraction of each sample period during which the detector in lane 3 output is “1”.
+  - **speed3 (double):**  The average speed of vehicles in lane 3 during the sample period.
+  - **flow4 (int):**  Null for this location. The number of vehicles observed in lane 4 for the sample period.
+  - **occupancy4 (double):** Null for this location. The fraction of each sample period during which the detector in lane 4 output is “1”.
+  - **speed4 (double):**  Null for this location. The average speed of vehicles in lane 4 during the sample period.
+  - **flow5 (int):**  Null for this location. The number of vehicles observed in lane 5 for the sample period.
+  - **occupancy5 (double):** Null for this location. The fraction of each sample period during which the detector in lane 5 output is “1”.
+  - **speed5 (double):**  Null for this location. The average speed of vehicles in lane 5 during the sample period.
+  - **flow6 (int):**  Null for this location. The number of vehicles observed in lane 6 for the sample period.
+  - **occupancy6 (double):** Null for this location. The fraction of each sample period during which the detector in lane 6 output is “1”.
+  - **speed6 (double):**  Null for this location. The average speed of vehicles in lane 6 during the sample period.
+  - **flow7 (int):**  Null for this location. The number of vehicles observed in lane 7 for the sample period.
+  - **occupancy7 (double):** Null for this location. The fraction of each sample period during which the detector in lane 7 output is “1”.
+  - **speed7 (double):**  Null for this location. The average speed of vehicles in lane 7 during the sample period.
+  - **flow8 (int):**  Null for this location. The number of vehicles observed in lane 8 for the sample period.
+  - **occupancy8 (double):** Null for this location. The fraction of each sample period during which the detector in lane 8 output is “1”.
+  - **speed8 (double):**  Null for this location. The average speed of vehicles in lane 8 during the sample period.
+
+# Source and License
+
+This data was built from data sets publicly available on [Caltrans Performance Measurement System (PeMS)](https://pems.dot.ca.gov/). It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
```

Comment:
A few questions:
1) How big is this data set?  I see 1425 files.
2) Is `./pems/parquet/pems/_SUCCESS` needed  It is an empty file?  If it is needed, what does it do?
3) Are `./pems/parquet/pems/_common_metadata` and `./pems/parquet/pems/_metadata` Deephaven created files?  Do they come from somewhere else?
4) For the files in 2 & 3 above, do we need to document anything about them in the readme?  I clearly don't know what they do, so I'm guessing others won't either.

Reply:
1) 43.2 MB for 1,430 items including _* and README
2) did not need _SUCCESS
3) These files are useful for reading, one of the reasons for switching to this data set is to talk about them in the docs and users can see the files, while the other data set did not have them.
4) added.